public/styles/main.css

```diff
@@ -115,3 +115,9 @@ input {
 #toolbar button {
     margin-right: 5px;
 }
+
+fieldset input {
+    width: auto;
+    height: auto;
+    float: left;
```

Comment:
do we need the float if it's in a `group`, which is `display:flex`? I feel like it might work without it